ix-dev/community/authelia/questions.yaml

```diff
@@ -28,6 +28,15 @@ questions:
     schema:
       type: dict
       attrs:
+        - variable: base_path
+          label: Base Path
+          description: |
+            If the path is configured to anything other than / requests will be handled for both / and the configured path.</br>
```

Comment:
if this should always start with `/` it should be matched to a regex.

Reply:
@stavros-k I don't know how to do that. Do you have doc or example?

Reply:
check other questions.yaml for `valid_chars` and `valid_chars_error`!